src/main/java/frc/robot/motion_profiling/Path.java

```diff
@@ -59,15 +63,29 @@ public Path(boolean reversed, double startVelocity, double endVelocity, Waypoint
         this.reversed = reversed;
         this.startVelocity = startVelocity;
         this.endVelocity = endVelocity;
-        
-        config = new TrajectoryConfig(robotConstants.motionProfilingConstants.MAX_VELOCITY, robotConstants.motionProfilingConstants.MAX_ACCELERATION)
-            .addConstraint(new CentripetalAccelerationConstraint(robotConstants.motionProfilingConstants.MAX_CENTRIPETAL_ACCELERATION))
-            .setKinematics(Robot.drivetrain.getKinematics())
-            .setReversed(reversed)
-            .setStartVelocity(startVelocity)
-            .setEndVelocity(endVelocity);
+        TrajectoryConfig config = new TrajectoryConfig(robotConstants.motionProfilingConstants.MAX_VELOCITY, robotConstants.motionProfilingConstants.MAX_ACCELERATION)
+                .addConstraint(new CentripetalAccelerationConstraint(robotConstants.motionProfilingConstants.MAX_CENTRIPETAL_ACCELERATION))
+                .setKinematics(Robot.drivetrain.getKinematics())
+                .setReversed(reversed)
+                .setStartVelocity(startVelocity)
+                .setEndVelocity(endVelocity);
 
-        trajectory = TrajectoryGenerator.generateTrajectory(Arrays.asList(waypoints), config); 
+        trajectory = TrajectoryGenerator.generateTrajectory(Arrays.asList(waypoints), config);
+    }
+
+    /**
+     * @param pathName the name of the path to load from the filesystem.
+     */
+    public Path(String pathName) {
+        var path = Paths.get(Filesystem.getDeployDirectory() +
+                "/paths/" + pathName);
+        try {
+            trajectory = TrajectoryUtil.fromPathweaverJson(path);
+        } catch (IOException e) {
+            System.err.println("could not load path from: " + path.toString()
+                    + " initializing with empty path instead of loading");
```

Comment:
System.err.println("could not load path from: " + path.toString()
Big C pls and print the path name first:
            System.err.println("Could not load " + pathName + " path from: " + path.toString()